BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -21,18 +25,64 @@ class ProfileViewController: UIViewController {
     @IBOutlet weak var imageProfile: UIImageView!
     @IBOutlet weak var labelPosition: UILabel!
     @IBOutlet weak var labelField: UILabel!
+    @IBOutlet weak var buttonLogout: UIButton!
     
     private var profileVM: ProfileViewModel?
+    private var member: ProfileData? {
+        didSet {
+            self.tableView.reloadData()
+        }
+    }
+    public var mode: screenType = .myProfile
+    public var memberId: Int?
+    
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.setUpTable()
         self.profileVM = ProfileViewModel()
-        self.setUpProfile()
-        self.updateProfileData() 
+        
+        switch mode {
+        case .myProfile:
+            self.updateProfileData()
+        case .teamMember:
+            self.memberConfiguration()
+            self.getMemberProfile()
+        }
+         
+    }
+    
+    // MARK:- memberConfiguration function
+    private func memberConfiguration() {
+        self.buttonLogout.isHidden = true
+    }
+    
+    // MARK:- getMemberProfile function
+    private func getMemberProfile() {
+        self.profileVM?.getMemberProfile(memberId, { result in
+            switch result {
+            case .success(let member):
+                self.member = member
+                self.setUpProfile()
+            case .failure(let error):
+                if error.asAFError?.responseCode == HttpStatusCode.unauthorized.rawValue {
```

Comment:
This is great.  What happens if error is not unauthorized error? do you let user knows that something was wrong?